src/indexes.ts

```diff
@@ -0,0 +1,101 @@
+import { indexer } from './indexer';
+
+module.exports = function (mcData: {
+  readonly biomes: readonly any[] | undefined;
```

Comment:
Too much duplicative code here. It might make sense to define this as an interface in a separate file so it doesn’t pollute the logic. Also, the type data here can probably be generated from the json schemas